api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -1587,4 +1569,47 @@ private void verifyOffsetAndLimit(Integer offset, Integer limit) {
             throw new IllegalArgumentException("Limit must be > 0");
         }
     }
-}
\ No newline at end of file
+
+    public PhoneNumbersResponse listNumbersForPurchase(String countryCode) throws IllegalArgumentException, GeneralException, UnauthorizedException, NotFoundException {
+        final String url = String.format("%s/v1/available-phone-numbers", NUMBERS_CALLS_BASE_URL);
```

Comment:
better to add v1 to NUMBERS_CALLS_BASE_URL. Later, updating version from one parameter would be easier.